corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -433,10 +432,6 @@ def generate_rows_and_filters(report_data_cache, report_config, restore_user, ro
     )
     filters_elem = _get_filters_elem(defer_filters, filter_options_by_field, restore_user._couch_user)
 
-    if (report_config.report_id in settings.UCR_COMPARISONS and
-        COMPARE_UCR_REPORTS.enabled(uuid.uuid4().hex, NAMESPACE_OTHER)):
```

Comment:
can you remove this FF as well

Reply:
260d92f1e72cc3d2f453d53d7fdd79c3523fca67